workflow/rules/assembly.smk

```diff
@@ -109,7 +109,7 @@ rule align_contigs:
         target="resources/genomes/main.fasta",
         query=get_quast_fastas,
     output:
-        "results/{date}/aligned/ref~main/{stage}~{sample}.bam",
+        temp("results/{date}/aligned/ref~main/{stage}~{sample}.bam"),
```

Comment:
maybe we should discuss if we keep one of the important alignments

Reply:
I don't how we can indicate, that only certain stages should be kept. Removed the temp flag in f095593.